### PR TITLE
Conclude `Catch the Beat Indonesia Cup 2021`

### PR DESCRIPTION
**Pull request by Nivalyx**

### CODE REVIEW

File: wiki/Tournaments/CIC/2021/en.md [hunk 1]
```diff
@@ -30,9 +30,11 @@ The **Catch the Beat Indonesia Cup 2021** (***CIC 2021***) is a double-eliminati
 | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of the raised prize pool, 2 months of osu!supporter |
 | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the raised prize pool, 1 month of osu!supporter |
 
+![Profile badge](img/badge.png)
```
Comment (TPGPL) [inline wiki/Tournaments/CIC/2021/en.md:33]:
```suggestion
![](img/badge.png "CIC 2021 winner badge")
```

File: wiki/Tournaments/CIC/2021/en.md [hunk 2]
```diff
@@ -53,6 +55,10 @@ CIC 2021 is run by various Indonesian osu! community members.
 - [Challonge brackets](https://challonge.com/CIC2021 "Challonge")
 - [Livestream channel](https://www.twitch.tv/osuIndonesia/ "Twitch")
 
+## Podium
+
+![Champion: dika312, 2nd place: CamXDanna, 3rd place: Nekuro](img/podium.png)
```
Comment (TPGPL) [inline wiki/Tournaments/CIC/2021/en.md:60]: move the image after the `Participants` header and remove `## Podium`

Comment (TPGPL) [reply wiki/Tournaments/CIC/2021/en.md:60]:
```suggestion
## Podium

![Champion: dika312, 2nd place: CamXDanna, 3rd place: Nekuro](img/podium.png)
```

Comment (TPGPL) [reply wiki/Tournaments/CIC/2021/en.md:60]:
```suggestion
## Podium

![](img/podium.png "CIC 2021 podium")
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Comment by Nivalyx** [comment]
ah, is moving the podium an absolute necessity? i prefer to keep it as is tho if you don't mind as i think it's more fitting to have the image in their own section...

also iirc the preceding iterations of the tournament have also featured the podium image on its own so yeah :

image

((applied everything else otherwise))

**Comment by TPGPL** [comment]
i guess its fine to have it in a seperate section then, but it still needs to be moved under `Participants`

**Review by TPGPL — APPROVED** [review]

**Pull request merged**